site/public/js/ta-grading.js

```diff
@@ -302,7 +302,7 @@ function notebookScrollLoad() {
     }
     if (element !== null) {
       if (element.length !== 0) {
-        notebookView.scrollTop(element.offset().top - notebookView.offset().top + notebookView.scrollTop());
+        notebookView.animate({scrollTop: (element.offset().top - notebookView.offset().top + notebookView.scrollTop())}, 100);
```

Comment:
Do you know why using `.scrollTop` was not working and `.animate` does? I would think both should work with the only difference being that the latter looks smoother.

Reply:
I just took a look at this. It seems it might be trying to scrollTop too early so it doesn't work. If you go in the JS console and call the notebookScrollLoad function it actually fixes itself. I was able to get it worked by moving the call to this function down a couple of lines. I'm not too familiar with the TA grading code so I don't know the best place to move the call to but I figure making this get called after the loading is done would probably be better than hoping the animate waits long enough.

Reply:
Thanks for checking - I tested some more and it looks like `adjustGradingPanelHeader` was causing the browser to reset the scroll position inside of the notebook view, so the delay was unneeded.